charts/camunda-platform-alpha-8.8/templates/connectors/configmap.yaml

```diff
@@ -36,23 +36,30 @@ data:
 
     camunda:
       client:
-        zeebe:
-          {{-  $proto := (lower .Values.core.readinessProbe.scheme) -}}
-          {{- $baseURLInternal := printf "%s://%s" $proto (include "core.fullname" . | trimAll "\"") }}
-          rest-address: {{ printf "%s:%v%s" $baseURLInternal .Values.core.service.httpPort .Values.core.contextPath }}
-          grpc-address: {{ printf "%s:%v" $baseURLInternal .Values.core.service.grpcPort }}
+        {{-  $proto := (lower .Values.core.readinessProbe.scheme) -}}
+        {{- $baseURLInternal := printf "%s://%s" $proto (include "core.fullname" . | trimAll "\"") }}
+        enabled: true
+        rest-address: {{ printf "%s:%v%s" $baseURLInternal .Values.core.service.httpPort .Values.core.contextPath }}
+        grpc-address: {{ printf "%s:%v" $baseURLInternal .Values.core.service.grpcPort }}
```

Comment:
❓ Do we need to allow users to configure `preferRestOverGrpc`?

Reply:
I would stick to the default for now for the connectors setup, can't see how a customer would care about this right now  🤔 wdyt?

Reply:
At the moment, no. I'm fine with sticking with the default.

I do know in the past there was talk about high performant use cases remaining on gRPC, such as job activation. But it's easy to add at that point.